go.mod

```diff
@@ -1,13 +1,13 @@
 module github.com/gobuffalo/buffalo
 
-go 1.15
+go 1.16
```

Comment:
@fasmat Am I right on thinking changing the version on the Go.mod to v1.16 will make buffalo compatible only with Go 1.16+ ? If so I think we should wait for v1.17 to be out to make this chance.

Reply:
Yes, the version statement indicates to go which is the oldest version of go that the module is compatible with (since newer versions will compile it anyway). I will revert it back to 1.15.